core/src/main/java/discord4j/core/object/entity/BaseGuildMessageChannel.java

```diff
@@ -214,7 +214,7 @@ public int getRawPosition() {
      * @return {@code true} if this channel is considered NSFW (Not Safe For Work), {@code false} otherwise.
      */
     public boolean isNsfw() {
-        return getData().isNsfw();
+        return getData().isNsfw() != null && getData().isNsfw();
```

Comment:
Its not better check or fix this in the setdata from channel or use a Optional::orElse?